UKHomeOffice/sgmr-service
# SPCR-511 - Add getVoyageData to customs page to enable prefill

### AC / Description of changes

getData function was missing on customs page meaning neither customs nor skipper were prefilling correctly

### To Test

- fill out a voyage form (don't submit)
- go to check your answers page
- click on 'change' for people page
- continue, ensure customs page is prefilling
- ensure skipper page is prefilling


### Notes

---

**Remember to merge to feature branches not `master`.**
